src/lib.rs

```diff
@@ -982,6 +1023,12 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_size_of_node() {
+        assert_eq!(std::mem::size_of::<CLruNode<String, usize>>(), 24);
+        assert_eq!(std::mem::size_of::<Option<CLruNode<String, usize>>>(), 24);
+    }
```

Comment:
That passes? Interesting. However, we probably do not want to make assumption like this as it will fail on non-64bit platforms and when the memory layout of structs changes.